x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/type/DataTypes.java

```diff
@@ -175,7 +175,7 @@ public static Integer metaSqlDataType(DataType t) {
     }
 
     // https://github.com/elastic/elasticsearch/issues/30386
-    // https://docs.microsoft.com/en-us/sql/odbc/reference/syntax/sqlgettypeinfo-function?view=sql-server-2017
+    // https://docs.microsoft.com/en-us/sql/odbc/reference/syntax/sqlgettypeinfo-function
     public static Integer metaSqlDateTimeSub(DataType t) {
         if (t == DATETIME) {
```

Comment:
Also here: `isDateBased()`?

Reply:
this was skipped but shouldn't be changed too?

Reply:
I don't think so since the type info indicates it should be just for datetime (timestamp) not DATE or TIME